### PR TITLE
Jetpack: Fix undefined variable warnings in featured images fallback

### PR DESCRIPTION
**Pull request by anomiex**
## Proposed changes:

PR #32824 introduced some new code, and unfortunately some code paths do not define variables that are needed later on. The later code handles the resulting `null`s properly, but we should still avoid the warnings.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

p1695140087285159/1695132622.239659-slack-C01U2KGS2PQ

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

* Run the tests from #32824, checking the log for warnings?

### CODE REVIEW

File: projects/plugins/jetpack/modules/theme-tools/content-options/featured-images-fallback.php [hunk 1]
```diff
@@ -63,9 +63,10 @@ function jetpack_featured_images_fallback_get_image( $html, $post_id, $post_thum
 
 			$image_sizes = '';
 
+			$width  = isset( $image['width'] ) ? intval( $image['width'] ) : null;
+			$height = isset( $image['height'] ) ? intval( $image['height'] ) : null;
+
 			if ( $image['src_width'] && $image['src_height'] && $image['width'] && $image['height'] ) {
```
Comment (zinigor) [inline projects/plugins/jetpack/modules/theme-tools/content-options/featured-images-fallback.php]: We also have an
```
Undefined array key "src_width"
```
on this line, should we add an `isset( $image['src_width'] )` here?

Comment (anomiex) [reply projects/plugins/jetpack/modules/theme-tools/content-options/featured-images-fallback.php]: Probably more like
```suggestion
			if ( ! empty( $image['src_width'] ) && ! empty( $image['src_height'] ) && ! empty( $image['width'] ) && ! empty( $image['height'] ) ) {
```

**Review by zinigor — APPROVED** [review]
Thanks for the quick fix!

**Comment by sgomes** [comment]
Thank you for the fix, @anomiex, and I apologise for the oversight 😞

**Pull request merged**